src/credentials/CHIPCert.h

```diff
@@ -627,6 +633,8 @@ CHIP_ERROR DecodeChipDN(chip::TLV::TLVReader & reader, ChipDN & dn);
 CHIP_ERROR ConvertX509CertToChipCert(const uint8_t * x509Cert, uint32_t x509CertLen, uint8_t * chipCertBuf,
                                      uint32_t chipCertBufSize, uint32_t & chipCertLen);
 
+CHIP_ERROR ExtractPubkeyFromX509Cert(const uint8_t * x509Cert, uint32_t x509CertLen, Crypto::P256PublicKey & pubkey);
```

Comment:
What is the use case for this function? Maybe more specifically, why there is a need to extract public key from X509 cert but not from CHIP Cert?